backend/app.py

```
@@ -27,6 +30,17 @@
 def health_check():
     return {"message": "서버가 정상 실행중입니다."}
 
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    """
+    서버 생명주기 관리
 
+    Args:
+        app (FastAPI): FastAPI 애플리케이션
+    """
+    asyncio.create_task(_save_history_to_db())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)